homeassistant/components/lovelace/__init__.py

```diff
@@ -18,10 +18,24 @@
 LOVELACE_CONFIG_FILE = 'ui-lovelace.yaml'
 JSON_TYPE = Union[List, Dict, str]  # pylint: disable=invalid-name
 
+FORMAT_YAML = 'yaml'
+FORMAT_JSON = 'json'
+
 OLD_WS_TYPE_GET_LOVELACE_UI = 'frontend/lovelace_config'
 WS_TYPE_GET_LOVELACE_UI = 'lovelace/config'
+
 WS_TYPE_GET_CARD = 'lovelace/config/card/get'
 WS_TYPE_SET_CARD = 'lovelace/config/card/set'
```

Comment:
Should we rename `set` to `update`, maybe it is clearer?

Reply:
I like update.